### PR TITLE
odo release notes

### PR DESCRIPTION
**Pull request by yhontyk**
Branch 4.6

### CODE REVIEW

File: cli_reference/developer_cli_odo/odo-release-notes.adoc [hunk 1]
```diff
@@ -8,48 +8,22 @@ toc::[]
...
+* Added documentation about converting S2I to devfile. See, xref:../../cli_reference/developer_cli_odo/using-devfiles-in-odo.adoc#converting-an-s2i-component-into-a-devfile-component_creating-applications-by-using-devfiles[Converting an S2I component into a devfile component].
 
-* The `--now` flag is added for `odo url delete` when using the devfiles.
+* Added commands to display JSON output for devfile components. 
 
-* `odo url delete --now` now works with the devfiles.
-* The `--debug` flag now works with the devfiles.
-
-* Added machine-readable output for listing Operator-backed services. Run `odo catalog list services -o json` to display information about Operators and services in JSON format.
+* `odo debug` is is no longer a Technology Preview feature and is now fully supported.
```
Comment (dharmit) [inline cli_reference/developer_cli_odo/odo-release-notes.adoc:19]: Two `is`. Please remove one.

File: cli_reference/developer_cli_odo/odo-release-notes.adoc [hunk 2]
```diff
@@ -8,48 +8,22 @@ toc::[]
...
+To learn more, see xref:../../cli_reference/developer_cli_odo/debugging-applications-in-odo.html#debugging-an-application_debugging-applications-in-odo[Debugging applications in odo].
 
-* Added machine-readable output for debugging. Run `odo debug info -o json` to display the debugging information in JSON format.
+* `{odo-title}` integration wit Operators support. You can now create services from Operators with `{odo-title}` is no longer a Technology Preview feature and is now fully supported. To learn more, see xref:../../cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc[Creating instances of services managed by Operators].
```
Comment (dharmit) [inline cli_reference/developer_cli_odo/odo-release-notes.adoc:22]: s/wit/with

Comment (dharmit) [reply cli_reference/developer_cli_odo/odo-release-notes.adoc:22]: This doesn't sound right : ` You can now create services from Operators with odo is no longer a Technology Preview feature and is now fully supported. ` 

Something's missing as a punctuation maybe?

**Review by dharmit — CHANGES_REQUESTED** [review]

**Comment by dharmit** [comment]
LGTM.

**Review by kalexand-rh — COMMENTED** [review]
LGTM!

**Pull request merged**